backend/src/Controller/Admin/CaptainFinancialSystem/AdminCaptainFinancialSystemAccordingToCountOfOrdersController.php

```diff
@@ -0,0 +1,145 @@
+use App\Request\Admin\CaptainFinancialSystem\AdminCaptainFinancialSystemAccordingToCountOfOrdersCreateRequest;
+use App\Service\Admin\CaptainFinancialSystem\AdminCaptainFinancialSystemAccordingToCountOfOrdersService;
+use Nelmio\ApiDocBundle\Annotation\Security;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use stdClass;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use OpenApi\Annotations as OA;
+
+/**
+ * Create and fetch Captain's Financial System According To Count Of Orders.
+ * @Route("v1/admin/captainfinancialsystemaccordingntocountofordersbyadmin/")
+ */
+class AdminCaptainFinancialSystemAccordingToCountOfOrdersController extends BaseController
+{
+    private AutoMapping $autoMapping;
+    private ValidatorInterface $validator;
+    private AdminCaptainFinancialSystemAccordingToCountOfOrdersService $adminCaptainFinancialSystemAccordingToCountOfOrdersService;
+
+    public function __construct(SerializerInterface $serializer, AutoMapping $autoMapping, ValidatorInterface $validator, AdminCaptainFinancialSystemAccordingToCountOfOrdersService $adminCaptainFinancialSystemAccordingToCountOfOrdersService)
+    {
+        parent::__construct($serializer);
+        $this->autoMapping = $autoMapping;
+        $this->validator = $validator;
+        $this->adminCaptainFinancialSystemAccordingToCountOfOrdersService = $adminCaptainFinancialSystemAccordingToCountOfOrdersService;
+    }
+
+    /**
+     * admin:Captain's Financial System According To Count Of Orders
+     * @Route("captainfinancialsystemaccordingtocountofordersbyadmin", name="createCaptainFinancialSystemAccordingToCountOfOrdersByAdmin", methods={"POST"})
+     * @IsGranted("ROLE_ADMIN")
+     * @param Request $request
+     * @return JsonResponse
+     *
+     * @OA\Tag(name="Captain's Financial System According To Count Of Orders")
+     *
+     * @OA\Parameter(
+     *      name="token",
+     *      in="header",
+     *      description="token to be passed as a header",
+     *      required=true
+     * )
+     *
+     * @OA\RequestBody(
+     *      description="new Captain's Financial System According To Count Of Orders",
+     *      @OA\JsonContent(
+     *          @OA\Property(type="integer", property="countOrdersInMonth"),
+     *          @OA\Property(type="number", property="salary"),
+     *          @OA\Property(type="number", property="monthCompensation"),
+     *          @OA\Property(type="number", property="bounceMaxCountOrdersInMonth"),
+     *          @OA\Property(type="number", property="bounceMinCountOrdersInMonth")
+     *      )
+     * )
+     *
+     * @OA\Response(
+     *      response=201,
+     *      description="Returns Captain's Financial System According On Order",
```

Comment:
description field must be updated to: ... According on count of orders (not orders), right? :)